data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py

```diff
@@ -13,470 +13,414 @@
 #     name: python3
 # ---
 
-# + [markdown] papermill={"duration": 0.024011, "end_time": "2021-02-02T22:30:31.951734", "exception": false,
-# "start_time": "2021-02-02T22:30:31.927723", "status": "completed"} tags=[]
+# + [markdown] papermill={"duration": 0.024011, "end_time": "2021-02-02T22:30:31.951734", "exception": false, "start_time": "2021-02-02T22:30:31.927723", "status": "completed"} tags=[]
 # #  QA queries on new CDR_deid COPE Survey
 #
 # Quality checks performed on a new CDR_deid dataset using QA queries
 
-# + papermill={"duration": 0.709639, "end_time": "2021-02-02T22:30:32.661373", "exception": false,
-# "start_time": "2021-02-02T22:30:31.951734", "status": "completed"} tags=[]
-
+# + papermill={"duration": 0.709639, "end_time": "2021-02-02T22:30:32.661373", "exception": false, "start_time": "2021-02-02T22:30:31.951734", "status": "completed"} tags=[]
+import urllib
 import pandas as pd
+from common import JINJA_ENV
 from utils import auth
 from gcloud.bq import BigQueryClient
 from analytics.cdr_ops.notebook_utils import execute, IMPERSONATION_SCOPES
 pd.options.display.max_rows = 120
 
-# + tags=["parameters"]
+# + papermill={"duration": 0.023643, "end_time": "2021-02-02T22:30:31.880820", "exception": false, "start_time": "2021-02-02T22:30:31.857177", "status": "completed"} tags=["parameters"]
+# Parameters
 project_id = ""
 com_cdr = ""
 deid_cdr = ""
-deid_sandbox = ""
-run_as = ""
-# deid_base_cdr=""
-# -
+#deid_sandbox=""
+sandbox=""
+run_as=""
 
-# df will have a summary in the end
-df = pd.DataFrame(columns=['query', 'result'])
+# +
+# Parameters
+
+#project_id = "aou-res-curation-prod"
```

Comment:
This parameter should be removed, we do not want our dataset and project names in the git history.